ivy/functional/frontends/paddle/tensor/math.py

```diff
@@ -346,3 +346,13 @@ def frac(x, name=None):
 @to_ivy_arrays_and_back
 def max(x, axis=None, keepdim=False, name=None):
     return ivy.max(x, axis=axis, keepdims=keepdim)
+
+@with_supported_dtypes(
+    {"2.5.0 and below": ("float32", "float64", "int32", "int64")},
+    "paddle",
+)
+@to_ivy_arrays_and_back
+def trace(x, offset=0, axis1=0, axis2=0, name=None):
+    return ivy.trace(x)
+
+
```

Comment:
Please follow the formatting convention of 2 blank lines between function definitions and a newline at end-of-file.